test/command_callback/test_racket_raco_command_callback.vader

```diff
@@ -0,0 +1,10 @@
+Before:
+  call ale#assert#SetUpLinterTest('racket', 'raco')
+
+After:
+  call ale#assert#TearDownLinterTest()
+
+Execute(The default command and executable should be correct):
+  AssertLinter 'raco', 'raco expand %s'
```

Comment:
I initially just plugged my code from what was in the docs. However, the ale#Escape function wasn't working for me. 
I assumed that this was because I was calling `raco expand %s` directly in `ale#linter#Define` under "command".

```
Execute(The default command should be correct):                                                                                                                                                                   
    " AssertLinter checks the executable and command.                                                                                                                                                               
    " Pass expected_executable, expected_command                                                                                                                                                                    
    AssertLinter 'some-command', ale#Escape('some-command') . ' --foo'
```

Reply:
`ale#Escape` is used for a _single_ file, not multiple separate arguments. In this case, it would be `ale#Escape("raco")`

Reply:
Yeah I tried that first. 
`AssertLinter 'raco', ale#Escape('raco') . ' expand %s'`

Unfortunately, that gives me an error. Is there something I'm doing wrong?